service/tasks/monitoring.py

```diff
@@ -493,25 +493,6 @@ def get_shared_identities(account_driver, cloud_machine, tenant_id_name_map):
     return identity_list
 
 
-def update_membership(application, shared_identities):
```

Comment:
deleted update membership
1) it is not being used
2) I think they meant to iterate through the shared_identities - for each identity in shared_identity, but they didn't do that.
3) I can't image this really being used for anything at this point.